versions.tf

```
@@ -1,9 +1,9 @@
 terraform {
-  required_version = ">= 0.13"
+  required_version = ">= 1.3.7"
   required_providers {
     scaleway = {
       source  = "scaleway/scaleway"
-      version = ">= 2.7.0"
+      version = ">= 2.17.0"
```

I'm not sure there's a lot of changes on bucket related resources since v2.10.0. Should be:
```suggestion
      version = ">= 2.10.0"
```